.coafile: Add YAMLLintBear support

akshkr opened the pull request:
This support ensures checking yml files using YAMLLintBear

Closes https://github.com/coala/coala-gedit/issues/19

Makman2 commented on .coafile:
```diff
@@ -8,3 +8,7 @@ allow_trailing_whitespace = False
+
+[ymlfiles]
+bears = YAMLLintBear
+files = **.yml
```
missing newline at EOF

Makman2 commented:
Commit message:
1. I think you can remove "support" from the shortlog. Just ".coafile: Add YAMLLintBear" is good imo :)
2. As you modified yml-files to adhere to the now enforced standards, I would recommend to describe this fact inside the commit body :)

akshkr committed:
.coafile: Add YAMLLintBear

This modification ensures checking yml files using YAMLLintBear
yml files are modified to adhere to the new enforced standards

Closes https://github.com/coala/coala-gedit/issues/19

khanchi97 reviewed (approved):
Changes LGTM :+1:

khanchi97 commented:
ack 6946042c

Makman2 commented:
@rultor merge

rultor commented:
> @rultor merge

@makman2 OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/13097-334938981)

rultor commented:
> @rultor merge

@makman2 Done! FYI, the full log is [here](http://www.rultor.com/t/13097-334938981) (took me 2min)
